middlewares/inputValidator.js

```diff
@@ -0,0 +1,37 @@
+
+
+/** * Class validating user inputs */
+export default class userValidator {
+  /**
+  * validates email
+  * @callback next
+  * @param {object} req - Request object
+  * @param {Object} res - Response object
+  * @param {next} next - Next route handler
+  * @returns {Object} failure message
+  */
+  static validateEmail(req, res, next) {
```

Comment:
And this as well. the validate class handles the validations properly

Reply:
My implementation may likely be better because it makes use of _express validator_  and achieved email validation with a line of code. I think that's more efficient. Besides, regex seems to have a more complex logic as compared to express validation logic. @andela-aonifade can you please look into this please, thanks

Reply:
Even if you are using express-validator, then you should be using that to replace the already existing method for email validation. The question of whether it is better will be discussed during our next standup.

Reply:
Going through the documentation, I found out the express validator you mentioned makes use of the Joi library which uses regex as well.

Reply:
Alright, thanks for finding time to review